Jenkinsfile

```diff
@@ -102,7 +102,7 @@ def runIntegrationTest(String description, String kubeprodArgs, String ginkgoArg
 
                 sh "kubectl --namespace kube-system get po,deploy,svc,ing"
 
-                sh "./bin/kubeprod -v=1 install --config=kubeprod-autogen.json ${kubeprodArgs}"
+                sh "./bin/kubeprod -v=1 install --manifests=manifests --config=kubeprod-autogen.json ${kubeprodArgs}"
```

Comment:
Maybe we should use a different name for --config to exercise generating the JSON configuration file under a different name. Something like --config=kubeprod.json?

Reply:
makes sense. I'll do it in a different PR